src/pgduckdb_background_worker.cpp

```diff
@@ -399,12 +407,12 @@ CreateSchemaIfNotExists(const char *postgres_schema_name, bool is_default_db) {
 	return true;
 }
 
-void
-SyncMotherDuckCatalogsWithPg_Unsafe(bool drop_with_cascade);
+void SyncMotherDuckCatalogsWithPg_Unsafe(bool drop_with_cascade);
```

Comment:
Was this change triggered by `make format`? It feels like it goes the other way on my end :-) Maybe I'm not up to date?

Reply:
Hmm, it seems I had a too new `clang-format` that did a bit more. Downgraded now for consistency to the version that is in our `dev_requirements.txt`. It doesn't seem to be formatted back with this older version though. So I'll leave it as is.